.github/workflows/changelog.yml

```diff
@@ -188,3 +188,6 @@ jobs:
           changeLogPath: xtensa-lx-rt/CHANGELOG.md
```

Comment:
Is there any point in keeping these checks now?

Reply:
Yes, the xtask doesn't actually check that the changelog changed